README.md

```diff
@@ -45,8 +45,8 @@ npm i @cityofzion/neon-js
 ### Nodejs
 
 ```js
-const Neon = require("@cityofzion/neon-js");
-const acct = Neon.create.Account("ALq7AWrhAueN6mJNqk6FHJjnsEoPRytLdW");
+import Neon from "@cityofzion/neon-js";
```

Comment:
If we are using `import`, there is an alternative you can consider:

```
import {default as Neon} from "@cityofzion/neon-js";
const acct = Neon.create.account("NKuyBkoGdZZSLyPbJEetheRhMjeznFZszf");
```

I think my suggestion (named imports) is the recommended style going forward.